packages/core/lib/test.js

```diff
@@ -213,8 +213,14 @@ const Test = {
     const updated =
       (await Profiler.updated(config.with({ resolver: testResolver }))) || [];
 
+    let compiler;
+    if (config.compileNone || config["--compile-none"]) {
+      compiler = "none";
+    }
```

Comment:
This will clobber if someone passes `--compiler` directly (not sure why they'd want that, but still...)

What if you did this instead:
```suggestion
    const compiler = (config.compileNone || config["--compile-none"])
      ? "none"
      : config.compiler;
```

Reply:
Yeah, that is nice.